src/OracleService/Protocols/OracleNeoFSProtocol.cs

```diff
@@ -75,14 +75,14 @@ private Task<string> GetAsync(Uri uri, string host, CancellationToken cancellati
             var tokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellation);
             tokenSource.CancelAfter(Settings.Default.NeoFS.Timeout);
             if (ps.Length == 2)
-                return GetPayloadAsync(client, objectAddr, tokenSource.Token);
-            return ps[2] switch
+                return await GetPayloadAsync(client, objectAddr, tokenSource.Token);
```

Comment:
Missing `await` will let client dispose immediately here, which we want it dispose after function `GetPayloadAsync` complete.

Reply:
I don't think so. I think something must be wrong elsewhere in the code.

Reply:
I created a script to show this small difference, https://dotnetfiddle.net/YiEMA7

Reply:
> I don't think so. I think something must be wrong elsewhere in the code.

That's true. We've tested it.